tests/common.py

```diff
@@ -212,14 +202,17 @@ def _retry_on_504_on_non_prod(func):
     for i in range(1, MAX_ATTEMPTS + 1):
         try:
             response = func()
-            if len(response.outputs) > 0 and response.outputs[0].status.code != status_code_pb2.RPC_REQUEST_TIMEOUT: # will want to retry
+            if (
```

Comment:
Is the outer parenthesis needed here?

Reply:
Yes, they are needed.  `If` expects expression started on the same line.  These parentheses are part of the expression, and allow to break it into multiple lines.